single-node/user-data

```diff
@@ -802,11 +870,14 @@ EOF
 [Unit]
 Requires=flanneld.service
 After=flanneld.service
+[Service]
+ExecStart=
+ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// \$DOCKER_OPTS \$DOCKER_CGROUPS \$DOCKER_OPT_MTU
```

Comment:
Rather than modify docker's `ExecStart` (could cause pain if upstream updates theirs), I'd rather just delete the code above that ends up creating the files "/etc/flannel/options.env /run/flannel/options.env" (and the EnvironmentFile reference for the latter).

That way docker will still have $DOCKER_BIP but it will simply expand to being empty.